sasha gong three entries second attempt

LenaHenningsen opened the pull request:
> Pull-requests should be ready to merge, that is free of conflicts, and they should pass our testsuite. If you require a specific reviewer you can request a review on the right.

### Please indicate issues related to this PR (if any)
See # .
Close # .

### Checklist
-   The title line explains what these changes do
-   All my changes are marked in the `last_modfied` column
-   The PR has no merge conflicts
-   Travis checks are passing

LenaHenningsen commented:
Hi there, ready to review, I guess - even though I vaguely think I forgot something. But what?

duncdrum commented:
Travis will tell you where the problems are [here](https://travis-ci.com/readchina/ReadingData/builds/99274518)

-   you dropped the leading zeros of dates in person.csv [5299](https://travis-ci.com/readchina/ReadingData/builds/99274518#L5299)
-   '101' is not a valid month [5239](https://travis-ci.com/readchina/ReadingData/builds/99274518#L5239) entered a month 101 on your last date change
-   there are empty columns and some errors likely due to not escaping fields with a comma [2334](https://travis-ci.com/readchina/ReadingData/builds/99274518#L2334)
-   some coordinates are missing a `.` in place [5382](https://travis-ci.com/readchina/ReadingData/builds/99274518#L5382)

LenaHenningsen commented:
Hi Duncan,

thanks for your mail.
The part about the leading zeros I don’t get, I only added missing info about sasha gong but made no changes otherwise. So I have no clue what this means.
101 – cleaned up (or so it seems to me)
In the acts table (empty columns) I added the last three lines, but no commas or semicolons, so this note is Buchstabensuppe to me, unfortunately
No idea why I got that as that was in the earlier entries – I corrected these; landport I have no idea though, will open an issue on that.
Anyhow, I’ll give it a second try now 😉

 

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Donnerstag, 31. Januar 2019 um 14:45
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Author <author@noreply.github.com>
Betreff: Re: [readchina/ReadingData] sasha gong three entries second attempt (#349)

 

Travis will tell you where the problems are here
you dropped the leading zeros of dates in person.csv 5299
'101' is not a valid month 5239 entered a month 101 on your last date change
there are empty columns and some errors likely due to not escaping fields with a comma 2334
some coordinates are missing a . in place 5382
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
